pkg/network/tracer/testutil/kafka/client.go

```diff
@@ -0,0 +1,100 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package kafka
```

Comment:
FYI this is marked as codeowned'd by networks, but should probably be usm.

Reply:
indeed, and I made the changes to the `CODEOWNERS` file in that PR
however, the content of the `CODEOWNERS` in the `main` branch is the one to determine for the PR who is the owner 😞